src/components/Users/UserAvailabilityTab.tsx

```
@@ -0,0 +1,330 @@
+                        </h3>
+                      </div>
+
+                      <div className="pl-5 py-2 space-y-4">
+                        {template.availabilities.map(
+                          ({
+                            name,
+                            slot_type,
+                            availability,
+                            slot_size_in_minutes,
+                          }) => (
+                            <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)